filebeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -271,3 +271,71 @@ _Kubernetes is not responsible for rotating logs, but rather a deployment tool s
 Different logrotation strategies can cause issues that might make Filebeat losing events or even duplicating events.
 Users can find more information about Filebeat's logrotation best practises at Filebeat's
 <<file-log-rotation,log rotation specific documentation>>
+
+[float]
+==== Kubernetes metadata
+
+In case filebeat is configured with kubernetes autodiscover provider (suggested configuration), kubernetes metadata
```

Comment:
```suggestion

===== *Case 1:** Filebeat with {k8s} provider

In case **filebeat is configured with kubernetes autodiscover provider** (suggested configuration), kubernetes metadata
```